lib/rentlinx/client.rb

```diff
@@ -96,8 +96,7 @@ def response_handler(response)
       when 204
         nil # don't attempt to JSON parse emptystring
       when 400
-        body = JSON.parse(response.body)
-        raise Rentlinx::BadRequest, response, body['details']
+        raise Rentlinx::BadRequest, response
```

Comment:
It seems that the 3-param version is not normally used.
It's for setting a custom traceback I think:
http://danielchangnyc.github.io/blog/2013/10/23/throw-raise/